src/Illuminate/Auth/Events/LogoutOtherDevices.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Illuminate\Auth\Events;
+
+use Illuminate\Queue\SerializesModels;
+
+class LogoutOtherDevices
+{
+    use SerializesModels;
+
+    /**
+     * The authentication guard name.
+     *
+     * @var string
+     */
+    public $guard;
+
+    /**
+     * The authenticated user.
+     *
+     * @var \Illuminate\Contracts\Auth\Authenticatable
+     */
+    public $user;
+
+    /**
+     * Create a new event instance.
+     *
+     * @param  string  $guard
+     * @param  \Illuminate\Contracts\Auth\Authenticatable  $user
+     * @return void
+     */
+    public function __construct($guard, $user)
```

Comment:
Is there a reason that the user parameter here and in the `fireLogoutOtherDevicesEvent` method isn't actually typehinted?

Reply:
None of the existing Auth event has a type-hint. If you want that feel free to make a different PR to suggest those changes. I don't want to bloat this PR to add huge changes just to be rejected.